setup.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020-2020 Vrije Universiteit Brussel
+#
+# This file is part of vsc-accounting-brussel,
+# originally created by the HPC team of Vrij Universiteit Brussel (http://hpc.vub.be),
+# with support of Vrije Universiteit Brussel (http://www.vub.be),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/sisc-hpc/vsc-accounting-brussel
+#
+# vsc-accounting-brussel is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation v2.
+#
+# vsc-accounting-brussel is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with vsc-manage.  If not, see <http://www.gnu.org/licenses/>.
+#
+##
+"""
+vsc-accounting-brussel base distribution setup.py
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+"""
+
+import vsc.install.shared_setup as shared_setup
+from vsc.install.shared_setup import ad
+
+PACKAGE = {
+    'version': '1.0.0',
```

Comment:
if you want to avoid have the version in two place, have a look at who we handled it on myresources

Reply:
done in https://github.com/sisc-hpc/vsc-accounting-brussel/pull/1/commits/6476ca983a4e0e62388e84a0c16da55d916e79c4